recipe/meta.yaml

```diff
@@ -49,6 +49,8 @@ requirements:
     - xgboost ==1.7.1
     - dask-ml ==2022.5.27
     - openssl >1.1.0g
+    - optuna ==3.1.0
```

Comment:
Just a heads up, this workflow as in the notebooks it needs 
```
xgboost=1.7.3
optuna==3.1.0b0 #this is a nightly
```
and I believe we might need a newer version of dask and distributed. 
For reference this is the [env file](https://github.com/coiled/dask-xgboost-nyctaxi/blob/main/environment.yml) that Greg used

Reply:
I ran the notebooks just fine with coiled-runtime 0.2.1 plus optuna 3.1.0 though.

Reply:
Ok then, maybe this was for the threading things. I remember @phobson mentioning this was needed.